packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/dashboard/dedicatedCloud-datacenter-dashboard.html

```diff
@@ -178,16 +178,5 @@
                 </oui-tile-definition>
             </oui-tile>
         </div>
-        <!-- Advices -->
```

Comment:
Why removing it ? if no advices no display ?

Reply:
Advises are there. Its no longer required to display these devices. It's a request from comex.

Reply:
you said that advices are still there even if we are removing it ?
I mean that maybe we should have placeholder to display advices and enable/disable them using backend.

Reply:
What I mean is we no longer need to display these advices for PCC now and in the future. Its safe to delete these and code related to them. I have enabled feature availability for advices. Its possible to enable/disable advices at app level if required.